pages/record/_.vue

```diff
@@ -100,27 +114,18 @@
         />
       </b-col>
     </b-row>
-    <b-row class="mb-3">
-      <!-- TODO: remove when the carousel has come to town. -->
-      <b-col>
-        <h2>Media</h2>
-        <WebResources
-          :media="media"
-        />
-      </b-col>
-    </b-row>
   </b-container>
 </template>
 
 <script>
   import EntityCards from '../../components/entity/EntityCards';
   import MediaActionBar from '../../components/record/MediaActionBar';
   import AlertMessage from '../../components/generic/AlertMessage';
-  import WebResources from '../../components/record/WebResources';
```

Comment:
Given this component is no longer used, delete it?

Reply:
I'm going to leave the translations in for now as we may very well need them when implementing the carousel.